### ISSUE TITLE
Insufficient contrast of default highlighted menu item - accessibility issue

### ISSUE THREAD

**Issue by michael-milette** [trema-tech/moodle-theme_trema#100]
The foreground and background colours are the same.
![image](https://user-images.githubusercontent.com/3808579/147655720-afe35f3b-aaf2-43df-91ec-41e8db3cb5b2.png)
This is an accessibility and usability issue.

**Comment by promosyofertas** [comment]
Any color that you choose, it happens the same. Would be great if this were selectable...

**Comment by promosyofertas** [comment]
I partially solved adding in Raw SCSS:
.dropdown-item:hover {
    background-color: #999600 ;
}
I'm still trying to find how to change the color for the first one that is always highlighted (using the up/down arrows moves along the menu but with the original color).

**Comment by rmady** [comment]
Hello @promosyofertas,

@michael-milette told me this issue and I fixed but just for Moodle 4.0, that still in beta now. This issue was solved with the focus-within. I'll add to master ASAP.

This is the code:
```
.dropdown-item:hover,
    .dropdown-item:focus,
    .dropdown-item:focus-within {
        background-color: lighten($primary, 20%);
        color: choose-contrast-color($primary);
    }
```

Thank you for your help guys!

**Comment by michael-milette** [comment]
Thank you! Any chance it can be fixed for currently supported versions of Moodle?

**Comment by promosyofertas** [comment]
> Thank you! Any chance it can be fixed for currently supported versions of Moodle?

@michael-milette, if you add what @rmady posted in Raw SCSS box you can fix it. It's not perfect because is taking the base color for "painting" but it's a good solution.

@rmady, it would be great if this could be selectable in the theme...

**Comment by michael-milette** [comment]
This will be fixed in the next major release of Trema. In the meantime, please feel free to try the experimental "dev" branch.

Best regards,

Michael
